index.html

```diff
@@ -4633,7 +4633,37 @@ <h2>Access to Personally Identifiable Information</h2>
   <section id="changes" class="appendix">
     <h1>Recent Specification Changes</h1>
 
-    <h2 id="changes-from-recommendation-1.0">Changes from the 1.0 version of [[wot-architecture]]</h2>
+    <h2 id="changes-from-fpwd-1.1">Changes from the FPWD version</h2>
+    <ul>
+      <li>Move Toru Kawaguchi to former editors.</li>
```

Comment:
Added Michael McCool to editors.
There were some other changes in the editors section.